src/nemo_evaluator/observability/model_traffic.py

```
@@ -133,6 +133,16 @@ def _truncate(value: Any, limit: int) -> Any:
     return value
 
 
+def _capture_request_body_value(body: Any, max_content_chars: int) -> Any | None:
```

Ruff (0.15.21)

[warning] Dynamically typed expressions (typing.Any) are disallowed in `body`

(ANN401)